### ISSUE TITLE
Warnings during installation -- "unmet peer dependency"

### ISSUE THREAD

**Issue by duanehutchins** [heroku/heroku-repo#87]
Trying to install this plugin for the first time on a fresh heroku install, and I get:

```
Installing plugin heroku-repo... yarn add v1.10.1
Installing plugin heroku-repo... [1/4] Resolving packages...Installing plugin heroku-repo... 
[2/4] Fetching packages...Installing plugin heroku-repo... 
[3/4] Linking dependencies...Installing plugin heroku-repo... 
[4/4] Building fresh packages...Installing plugin heroku-repo... success 
    Saved 99 new dependencies.Installing plugin heroku-repo... 
    info Direct dependenciesInstalling plugin heroku-repo... 
    info All dependenciesInstalling plugin heroku-repo... 
        ├─ cacheable-request@2.1.4Installing plugin heroku-repo... 
        ├─ debug@3.2.6Installing plugin heroku-repo... 
        ├─ fs-extra@4.0.3Installing plugin heroku-repo... 
        ├─ has-symbol-support-x@1.4.2Installing plugin heroku-repo... 
        ├─ http-call@3.0.2Installing plugin heroku-repo... 
        ├─ isexe@2.0.0Installing plugin heroku-repo... 
        ├─ lodash.ary@4.1.1Installing plugin heroku-repo... 
        ├─ lodash.partial@4.2.1Installing plugin heroku-repo... 
        ├─ moment@2.22.2Installing plugin heroku-repo... 
        ├─ npm-run-path@2.0.2Installing plugin heroku-repo... 
        ├─ password-prompt@1.0.7Installing plugin heroku-repo... 
        ├─ pify@3.0.0Installing plugin heroku-repo... 
        ├─ responselike@1.0.2Installing plugin heroku-repo... 
        ├─ shebang-command@1.2.0Installing plugin heroku-repo... 
        ├─ shellwords@0.1.1Installing plugin heroku-repo... 
        ├─ string_decoder@1.1.1Installing plugin heroku-repo... 
        ├─ tslib@1.9.3Installing plugin heroku-repo... 
        ├─ unzip-response@2.0.1Installing plugin heroku-repo... 
        ├─ url-parse-lax@3.0.0Installing plugin heroku-repo... 
        └─ which@1.3.1Installing plugin heroku-repo... Done in 3.22s.
Installing plugin heroku-repo... installed v1.0.12

warning "heroku-repo > heroku-run > cli-engine-heroku@4.1.1" has unmet peer dependency "cli-engine-command@^8.0.0".
warning "heroku-repo > heroku-run > cli-engine-heroku@4.1.1" has unmet peer dependency "cli-engine-config@^3.0.0".
```

**Comment by dmathieu** [comment]
cc @heroku/cli

**Comment by RasPhilCo** [comment]
Thanks! Fixed in `v1.0.13`.
